server/services/v1/database/search_reader.go

```diff
@@ -16,12 +16,13 @@ package database
 
 import (
 	"context"
-	"github.com/tigrisdata/tigris/server/metrics"
+
 
```

Comment:
Is it two empty lines?

Reply:
Okay, make test only shifted the imports but didn't remove that line.